app/controllers/appointment_summaries_controller.rb

```diff
@@ -0,0 +1,65 @@
+class AppointmentSummariesController < ApplicationController
+  layout 'guides'
+
+  helper_method :headers
+  helper_method :appointment_type
+
+  def new
+    @appointment_summary = AppointmentSummary.new
+  end
+
+  def create
```

Comment:
We're not actually creating anything so I'm not sure this the correct verb to use?

We're requesting an appointment summary, either in HTML or PDF and we need to supply specific params for the request to be accepted.